.drone.yml

```diff
@@ -5000,6 +5000,135 @@ volumes:
 - name: dockersock
   temp: {}
 
+---
+################################################
+# Generated using dronegen, do not edit by hand!
+# Use 'make dronegen' to update.
+# Generated at dronegen/misc.go:123
+################################################
+
+kind: pipeline
+type: kubernetes
+name: migrate-apt-new-repos
```

Comment:
Does this need to be a part of the drone pipeline?

Reply:
Thought about this a lot while building this part out. Putting it in the pipeline is by far the easiest way to implement migrations for older versions. In 99.9% of cases this pipeline will never be ran, but when we need to add previous versions this will save significant time. This section along with [this new dronegen function](https://github.com/gravitational/teleport/pull/10746/files/fe84b4577168e189b30ace973a3567adb43590dc#diff-740f521aa49a0d8dea41dcfcaa233a335a6543437ad14d6dd34cc2779230695aR35) takes another migration process like I did last week and turns it into a 5m change.

Reply:
I don't understand why this particular pipeline is here, though. I get `publish-apt-new-repos`, but not this one. Is it just marking that you intend to implement the migration of legacy here?